src/test/test-set.c

```diff
@@ -406,4 +406,48 @@ TEST(set_fnmatch) {
         assert_se(!set_fnmatch(match, nomatch, "cccXX"));
 }
 
+TEST(set_make_nulstr) {
+        _cleanup_set_free_ Set *set = NULL;
+        size_t len = 0;
+        int r;
+
+        {
+                // Unallocated and empty set.
```

Comment:
Coding style mandates to always use `/* */` style comments so please update these to use that.